src/main/java/org/venice/beachfront/bfapi/controllers/OAuthController.java

```diff
@@ -104,7 +104,7 @@ public String oauthLogout(HttpServletResponse response, Authentication authentic
 		userProfileService.invalidateKey(userProfileService.getProfileFromAuthentication(authentication));
 
 		// Construct redirect url for server side logout
-		final String uiUrl = "beachfront." + domain;
+		final String uiUrl = "http://beachfront." + domain;
```

Comment:
Shouldn't this be https? Is this something that will differ in environments?

Reply:
haha yes, and that was addressed in Waymond's email.  I was initially concerned about this (and i don't know why they didn't/couldn't just register it as https but that's an approved url they put on our whitelist for logout redirecting.  It ends up taking you to https anyhow, so i guess it's not that big a deal?